generate-release-vars.sh

```diff
@@ -3,9 +3,10 @@ set -eio pipefail
 
 usage() {
     echo "Usage:"
-    echo "  $0 ECS_INIT_VERSION AMI_VERSION"
```

Comment:
Do we have an ETA for a follow-up PR to include `ECS_INIT_VERSION` in project Shinkansen ? I would prefer to keep `ECS_INIT_VERSION` in the script, and only remove it in the follow-up PR.

Reply:
ETA is next week. It will be added back with a slightly different mechanism, but this requires file changes appended to @singholt's changes from the previous week's refactoring.